src/modules/SX128x/SX128x.h

```diff
@@ -84,6 +84,7 @@
 #define RADIOLIB_SX128X_REG_FREQ_ERROR_CORRECTION               0x093C
 #define RADIOLIB_SX128X_REG_LORA_SYNC_WORD_MSB                  0x0944
 #define RADIOLIB_SX128X_REG_LORA_SYNC_WORD_LSB                  0x0945
+#define RADIOLIB_SX128X_REG_LORA_RX_CODING_RATE                 0x0950
```

Comment:
Similar queestion as for the SX126x driver - where is this adress from?

Reply:
It’s from the latest version of the SX1280 datasheet (Rev. 3.3.), page 102.